### ISSUE TITLE
DNS-Server

### ISSUE THREAD

**Issue by Preisschild** [l-n-s/wireguard-install#4]
The client didn't had an dns server set. IMO the script should ask on first setup which dns server to use and put it in DNS = at the client configs.

**Comment by vjanssens** [comment]
Thanks to this issue, I knew I had to set DNS in the client config to actually get it working.
I don't know about best practices, but this was what I ended up with:

```
[Interface]
< some more settings >
DNS = 1.1.1.1
```

**Comment by Preisschild** [comment]
Yea, I've updated the script to automatically do this, but I hadn't the time yet to make it so that it asks the user the first time to set a dns server

**Comment by mgdudas** [comment]
Yeah, I'm going to chock this up to an iOS issue. I've reached out to wireguard to see if we can figure it out.

**Comment by Preisschild** [comment]
Hmm strange. I've tested it in Android and it seems to be working.

**Comment by EricKuck** [comment]
The iOS QR code implementation is working fine. The issue is with this script, which places DNS in the peer section rather than the interface section.
